projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/jobData.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This enum is automatically generated from the enum from  ../vdk-jupyterlab-extension/ui_job_properties.py
+ * Using https://github.com/Syndallic/py-to-ts-interfaces#example
+ */
+export enum JobProperties {
+  name = 'jobName',
+  team = 'jobTeam',
+  restApiUrl = 'restApiUrl',
+  path = 'jobPath',
+  cloud = 'cloud',
+  local = 'local',
+  arguments = 'jobArguments'
+}
+
+/*
+ * In the front-end extension a global variable is introduces (jobData) which holds the information about the current job.
+ * The values of its properties are meant to be changed during a VDK operation and after the operation ends they need to be set to default.
+ */
+export var jobData = new Map<string, string>([
+  [JobProperties.name, ''],
```

Comment:
nit: could save ourselves some work (adding manually new entries) -> if the map is automatically populated by auto-iterating over the enum values, then set those automatically, with the default value accordingly.

Reply:
Done